shared/login/relogin/index.desktop.tsx

```diff
@@ -46,10 +48,10 @@ class Login extends React.Component<Props, State> {
   render() {
     const userRows = this.props.users
       .concat({hasStoredSecret: false, username: other})
-      .map(u => <UserRow user={u.username} key={u.username} />)
+      .map(u => <UserRow user={u.username} key={u.username} hasStoredSecret={u.hasStoredSecret} />)
 
     const selectedIdx = this.props.users.findIndex(u => u.username === this.props.selectedUser)
-
+    const needPassword = !this.props.hidePasswordBox
```

Comment:
@jzila how's this compromise?

Reply:
But now you need to derive that variable twice.

Reply:
hmm I think for some reason `needPassword` is much more acceptable to me than `showPasswordBox`. Renamed to that.